backend/src/services/secret-import/secret-import-service.ts

```diff
@@ -118,7 +118,7 @@ export const secretImportServiceFactory = ({
         importEnv: folder.environment.id,
         importPath: secretPath
       });
-      if (existingImport) throw new BadRequestError({ message: "Cyclic import not allowed" });
+      if (existingImport) throw new NotFoundError({ message: "Cyclic import not allowed" });
```

Comment:
This feels like a bad request from the user? They shouldn't be doing cyclic import